core/src/create.jl

```diff
@@ -624,33 +625,98 @@ end
 function User(db::DB, config::Config)::User
     static = load_structvector(db, config, UserStaticV1)
     time = load_structvector(db, config, UserTimeV1)
-    defaults = (; min_level = -Inf, active = true)
-    time_interpolatables = [:demand]
-    parsed_parameters, valid = parse_static_and_time(
-        db,
-        config,
-        "User";
-        static,
-        time,
-        time_interpolatables,
-        defaults,
-    )
+
+    static_node_ids, time_node_ids, node_ids, valid =
+        static_and_time_node_ids(db, static, time, "User")
 
     if !valid
-        error("Errors occurred when parsing User (node type) data.")
+        error("Problems encountered when parsing User static and time node IDs.")
     end
 
-    allocated = zeros(length(parsed_parameters.return_factor))
+    # The highest priority number given, which corresponds to the least important demands
+    maximum_priority =
+        max(reduce(max, static.priority; init = 0), reduce(max, time.priority; init = 0))
 
-    return User(
-        parsed_parameters.node_id,
-        parsed_parameters.active,
-        parsed_parameters.demand,
-        allocated,
-        parsed_parameters.return_factor,
-        parsed_parameters.min_level,
-        parsed_parameters.priority,
+    active = BitVector()
+    min_level = Float64[]
+    return_factor = Float64[]
+    interpolations = Vector{ScalarInterpolation}[]
+
+    errors = false
+    trivial_timespan = [nextfloat(-Inf), prevfloat(Inf)]
+    t_end = seconds_since(config.endtime, config.starttime)
+
+    # Create a dictionary priority => time data for that priority
+    time_priority_dict = Dict(
+        first(group).priority => StructVector(group) for
+        group in IterTools.groupby(row -> row.priority, time)
     )
+
+    for node_id in node_ids
+        first_row = nothing
+        demand = Vector{ScalarInterpolation}()
+
+        if node_id in static_node_ids
+            rows = searchsorted(static.node_id, node_id)
+            static_id = view(static, rows)
+            for p in 1:maximum_priority
```

Comment:
This will not go well if a modeler wan't to be sure to give the lowest priority and thus sets

```julia
julia> typemax(Int)
9223372036854775807
```

Perhaps it is better to find the `sorted(unique(priorities))`` and loop over that?